index.html

```diff
@@ -118,6 +118,19 @@ <h2><i>WebGL.</i> <i>Simple.</i> <i>Powerful.</i></h2>
 				<h2>Demos</h2>
 				<div class="gallery">
 					<article class="gallery-item" itemscope="" itemtype="http://schema.org/CreativeWork">
+						<a title="Try WAZANA.IO Game" class="gallery-item-link" href="https://wazana.io/"
```

Comment:
please add "3rd" in the article class :)

Reply:
Ok I added "3rd" in the article class.

On the conflict, seems like this is an indentation issue. I can't reproduce the conflict in my project between master and homepage-add-wazanaio. 

Can you tell me how to resolve it by choosing the master changes?